updater/post_processing/post_process_assignee.py

```diff
@@ -188,19 +200,178 @@ def precache_assignees(**kwargs):
 
 
 def create_canonical_assignees(**kwargs):
-    config = get_current_config(schedule='quarterly',**kwargs)
+    config = get_current_config(schedule='quarterly', **kwargs)
```

Comment:
Should this function also create the view `assignee` to link it to `assignee_YYYYMMDD`?

Reply:
Yup it does: 
    target_table = "assignee_{suffix}".format(suffix=suffix)
    create_sql = """
    CREATE TABLE `{target_table}` (
    `id` varchar(36) COLLATE utf8mb4_unicode_ci NOT NULL,
  `type` int(4) DEFAULT NULL,
  `name_first` varchar(96) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `name_last` varchar(96) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `organization` varchar(256) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `version_indicator` date NOT NULL DEFAULT '{version_indicator}',
  `created_date` timestamp NOT NULL DEFAULT current_timestamp(),
  `updated_date` timestamp NULL DEFAULT NULL ON UPDATE current_timestamp(),
  PRIMARY KEY (`id`),
  KEY `assignee_version_indicator_index` (`version_indicator`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
    """.format(version_indicator=version_indicator, target_table=target_table)
    view_query = """
    CREATE OR REPLACE VIEW assignee as select * from {target_table}
    """.format(target_table=target_table)